docs/pct/README.md

```diff
@@ -35,7 +35,7 @@ Templates currently come in 2 flavours: `project` and `item`.
 
 PCT is available through the `pct new` command.
 
-The `--list` or `-l` flag displays a list of locally available templates located in the default template location `$HOME/.pdk/pct/`. The list of templates is also available by calling `pct new` without flags.
+The `--list` or `-l` flag displays a list of locally available templates located in the default template location `$CWD/templates`. The list of templates is also available by calling `pct new` without flags.
```

Comment:
Shouldn't this be 'the current directory of the PCT executable' and not current working directory?

Reply:
That is a very good point...yeah, bad assumption on my part they'd be here.

Also, probably need instructions to unzip the archive